docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

```diff
@@ -146,47 +146,44 @@ When materialized, the `iris_kmeans_jupyter` asset will execute the notebook (`/
+Open the `/tutorial_template/__init__.py` file and add the following code:
 
 ```python
-# tutorial_template/repository.py
+# tutorial_template/__init__.py
 
+from dagster import load_assets_from_package_module, Definitions
 from dagstermill import ConfigurableLocalOutputNotebookIOManager
-from dagster import repository, with_resources
+
 from . import assets
 
-@repository
-def template_tutorial():
-    return
-        with_resources(
-            load_assets_from_package_module(assets),
-            resource_defs={
-                "output_notebook_io_manager": ConfigurableLocalOutputNotebookIOManager(),
-            },
-        ),
+defs = Definitions(
+    assets=load_assets_from_package_module(assets),
+    resources={
+        "output_notebook_io_manager": ConfigurableLocalOutputNotebookIOManager()
+    }
+)
+
 ```
 
 Let's take a look at what's happening here:
 
-- Using <PyObject object="with_resources"/>, we've provided resources to the assets in the repository. In this example, that's the <PyObject module="dagstermill" object="ConfigurableLocalOutputNotebookIOManager" /> resource.
+- Using <PyObject object="load_assets_from_package_module" />, we've imported all assets in the `assets` module. This approach allows any new assets we create to be automatically added to the Definitions instead of needing to manually add them one by one.
 
-  This I/O manager, bound to the `output_notebook_io_manager` key, is responsible for handling the storage of the notebook asset's resulting `.ipynb` file.
-
-- Using <PyObject object="load_assets_from_package_module" />, we've imported all assets in the `assets` module into the repository. This approach allows any new assets we create to be automatically added to the repository instead of needing to manually add them one by one.
+- We provided a dictionary of resources to the `resources` parameter. In this example, that's the <PyObject module="dagstermill" object="ConfigurableLocalOutputNotebookIOManager" /> resource.
 
-- Finally, we return the result of `with_resources` from the <PyObject object="repository" decorator /> decorated function. This allows Dagster to find the assets we've defined and instantiate the necessary resource when the asset is materialized.
+  This I/O manager, bound to the `output_notebook_io_manager` key, is responsible for handling the storage of the notebook asset's resulting `.ipynb` file.
 
 ---
 
 ## Step 4: Materialize the notebook asset
 
-Now that you've created an asset, a resource, and a repository, it's time to materialize the notebook asset! Materializing an asset runs the op it contains and saves the results to persistent storage.
+Now that you've created an asset, a resource, and a Definitions object, it's time to materialize the notebook asset! Materializing an asset runs the op it contains and saves the results to persistent storage.
 
 1. To start Dagit, Dagster's web interface, run the following in `/tutorial_template`:
```

Comment:
I know this isn't a part of your PR, but would you mind changing `Dagit` to `Dagster UI` in this guide?